frontend/src/components/TextInputComponent.vue

```diff
@@ -63,6 +64,10 @@ defineProps({
     required: false,
     default: false,
   },
+  /**
+   *
+   */
+  input: String,
```

Comment:
shouldn't we rename this to `modelValue` so we can just use `v-model` in other components instead of `v-model:input`?

Reply:
that didn't worked for me, i needed to write then `v-model:modelValue`, i will try it again, may one of my other problem, that i fixed in meantime, was the problem

Reply:
works actually now